iot-hub/Samples/device/DeviceReconnectionSample/readme.md

```diff
@@ -0,0 +1,56 @@
+# Device Reconnection Sample
```

Comment:
This sample shows the connection status code for handling these scenarios, but how does a customer make them happen to see the code in action?

Reply:
You mean, steps on how to simulate an error, so that the client does retry/ recover? I'll add some notes on that, sure.

Reply:
Yep, like cycle a device primary key, pull network cable, etc.